README.md

```diff
@@ -159,10 +164,11 @@ commercetools/commercetools-project-sync:3.3.0 -s all
    ```
    This will run the following sync modules in the given order:
  1. `Type` Sync and `ProductType` Sync and `States` Sync in parallel.
- 2. `Category` Sync.
- 3. `Product` Sync.
- 4. `CartDiscount` Sync.
- 5. `InventoryEntry` Sync.
+ 2. `TaxCategory` Sync.
+ 3. `Category` Sync.
+ 4. `Product` Sync.
+ 5. `CartDiscount` Sync.
+ 6. `InventoryEntry` Sync.
 
```

Comment:
Also include the docker command below. Just like the other sync types has already added,

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/139/commits/92ba3bb67a2777e0a7dca1b0d4260f80723284ea